### PR TITLE
[deliver] add support to detecting and uploading 6.7" (iPhone 14 Pro Max) screenshots

### LINKED ISSUE #20653: iPhone 14 Pro and Pro Max screenshots have a new screen size that is not supported by deliver
### New Issue Checklist

- [x] Updated fastlane to the latest version
- [x] I read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I read [docs.fastlane.tools](https://docs.fastlane.tools)
- [x] I searched for [existing GitHub issues](https://github.com/fastlane/fastlane/issues)

### Issue Description

It looks like support for the new iPhone 14 Pro and Pro Max screenshot size needs to be added to deliver. The new screenshot size specifications can be found here: https://help.apple.com/app-store-connect/#/devd274dd925

##### Command executed

bundle exec fastlane deliver --skip_binary_upload --skip_metadata --force --overwrite_screenshots --api_key_path <redacted>

##### Complete output when running fastlane, including the stack trace and command used
<details>
[09:52:08]: Successfully loaded '/Users/biggs/jenkins/workspace/Upload App Store Screenshots/StudyJapanese/Deliverfile' 📄

+----------------------------+-------------------------------+
|            Detected Values from './Deliverfile'            |
+----------------------------+-------------------------------+
| metadata_path              | AppStoreConnect/metadata      |
| screenshots_path           | AppStoreConnect/screenshots   |
| app_identifier             | com.serpentisei.studyjapanese |
| username                   | <redacted>            |
| team_name                  | Serpenti Sei LLC              |
| run_precheck_before_submit | false                         |
+----------------------------+-------------------------------+

[09:52:08]: Creating authorization token for App Store Connect API

+--------------------------------------+----------------------------------+
|                         deliver 2.209.1 Summary                         |
+--------------------------------------+----------------------------------+
| skip_binary_upload                   | true                             |
| skip_metadata                        | true                             |
| force                                | true                             |
| overwrite_screenshots                | true                             |
| api_key_path                         | <redacted>|
| username                             | <redacted>               |
| app_identifier                       | com.serpentisei.studyjapanese    |
| platform                             | ios                              |
| edit_live                            | false                            |
| use_live_version                     | false                            |
| metadata_path                        | AppStoreConnect/metadata         |
| screenshots_path                     | AppStoreConnect/screenshots      |
| skip_screenshots                     | false                            |
| skip_app_version_update              | false                            |
| sync_screenshots                     | false                            |
| submit_for_review                    | false                            |
| verify_only                          | false                            |
| reject_if_possible                   | false                            |
| phased_release                       | false                            |
| reset_ratings                        | false                            |
| team_name                            | Serpenti Sei LLC                 |
| run_precheck_before_submit           | false                            |
| precheck_default_rule_level          | warn                             |
| ignore_language_directory_validation | false                            |
| precheck_include_in_app_purchases    | true                             |
+--------------------------------------+----------------------------------+

[09:52:11]: 🚫 Invalid screenshots were detected! Here are the reasons:
[09:52:11]: 🚫 Error: AppStoreConnect/screenshots/de-DE/1_iphone67_1.AppStoreNihongoEntry.png - Invalid screen size (Actual size is 1290x2796. See the specifications to fix [https://help.apple.com/app-store-connect/#/devd274dd925)](https://help.apple.com/app-store-connect/#/devd274dd925)%1B[0m)
[09:52:11]: 🚫 Error: AppStoreConnect/screenshots/de-DE/2_iphone67_2.AppStoreKanjiFlashcard.png - Invalid screen size (Actual size is 1290x2796. See the specifications to fix [https://help.apple.com/app-store-connect/#/devd274dd925)](https://help.apple.com/app-store-connect/#/devd274dd925)%1B[0m)
[09:52:11]: 🚫 Error: AppStoreConnect/screenshots/de-DE/3_iphone67_3.AppStoreWordTriage.png - Invalid screen size (Actual size is 1290x2796. See the specifications to fix [https://help.apple.com/app-store-connect/#/devd274dd925)](https://help.apple.com/app-store-connect/#/devd274dd925)%1B[0m)
[09:52:11]: 🚫 Error: AppStoreConnect/screenshots/de-DE/4_iphone67_4.AppStoreOCR.png - Invalid screen size (Actual size is 1290x2796. See the specifications to fix [https://help.apple.com/app-store-connect/#/devd274dd925)](https://help.apple.com/app-store-connect/#/devd274dd925)%1B[0m)
[09:52:11]: 🚫 Error: AppStoreConnect/screenshots/de-DE/5_iphone67_5.AppStoreViewClipping.png - Invalid screen size (Actual size is 1290x2796. See the specifications to fix [https://help.apple.com/app-store-connect/#/devd274dd925)](https://help.apple.com/app-store-connect/#/devd274dd925)%1B[0m)
[09:52:11]: 🚫 Error: AppStoreConnect/screenshots/de-DE/6_iphone67_6.AppStoreWikipedia.png - Invalid screen size (Actual size is 1290x2796. See the specifications to fix [https://help.apple.com/app-store-connect/#/devd274dd925)](https://help.apple.com/app-store-connect/#/devd274dd925)%1B[0m)
[09:52:11]: 🚫 Error: AppStoreConnect/screenshots/en-US/1_iphone67_1.AppStoreNihongoEntry.png - Invalid screen size (Actual size is 1290x2796. See the specifications to fix [https://help.apple.com/app-store-connect/#/devd274dd925)](https://help.apple.com/app-store-connect/#/devd274dd925)%1B[0m)
[09:52:11]: 🚫 Error: AppStoreConnect/screenshots/en-US/2_iphone67_2.AppStoreKanjiFlashcard.png - Invalid screen size (Actual size is 1290x2796. See the specifications to fix [https://help.apple.com/app-store-connect/#/devd274dd925)](https://help.apple.com/app-store-connect/#/devd274dd925)%1B[0m)
[09:52:11]: 🚫 Error: AppStoreConnect/screenshots/en-US/3_iphone67_3.AppStoreWordTriage.png - Invalid screen size (Actual size is 1290x2796. See the specifications to fix [https://help.apple.com/app-store-connect/#/devd274dd925)](https://help.apple.com/app-store-connect/#/devd274dd925)%1B[0m)
[09:52:11]: 🚫 Error: AppStoreConnect/screenshots/en-US/4_iphone67_4.AppStoreOCR.png - Invalid screen size (Actual size is 1290x2796. See the specifications to fix [https://help.apple.com/app-store-connect/#/devd274dd925)](https://help.apple.com/app-store-connect/#/devd274dd925)%1B[0m)
[09:52:11]: 🚫 Error: AppStoreConnect/screenshots/en-US/5_iphone67_5.AppStoreViewClipping.png - Invalid screen size (Actual size is 1290x2796. See the specifications to fix [https://help.apple.com/app-store-connect/#/devd274dd925)](https://help.apple.com/app-store-connect/#/devd274dd925)%1B[0m)
[09:52:11]: 🚫 Error: AppStoreConnect/screenshots/en-US/6_iphone67_6.AppStoreWikipedia.png - Invalid screen size (Actual size is 1290x2796. See the specifications to fix [https://help.apple.com/app-store-connect/#/devd274dd925)](https://help.apple.com/app-store-connect/#/devd274dd925)%1B[0m)
[09:52:11]: 🚫 Error: AppStoreConnect/screenshots/es-MX/1_iphone67_1.AppStoreNihongoEntry.png - Invalid screen size (Actual size is 1290x2796. See the specifications to fix [https://help.apple.com/app-store-connect/#/devd274dd925)](https://help.apple.com/app-store-connect/#/devd274dd925)%1B[0m)
[09:52:11]: 🚫 Error: AppStoreConnect/screenshots/es-MX/2_iphone67_2.AppStoreKanjiFlashcard.png - Invalid screen size (Actual size is 1290x2796. See the specifications to fix [https://help.apple.com/app-store-connect/#/devd274dd925)](https://help.apple.com/app-store-connect/#/devd274dd925)%1B[0m)
[09:52:11]: 🚫 Error: AppStoreConnect/screenshots/es-MX/3_iphone67_3.AppStoreWordTriage.png - Invalid screen size (Actual size is 1290x2796. See the specifications to fix [https://help.apple.com/app-store-connect/#/devd274dd925)](https://help.apple.com/app-store-connect/#/devd274dd925)%1B[0m)
[09:52:11]: 🚫 Error: AppStoreConnect/screenshots/es-MX/4_iphone67_4.AppStoreOCR.png - Invalid screen size (Actual size is 1290x2796. See the specifications to fix [https://help.apple.com/app-store-connect/#/devd274dd925)](https://help.apple.com/app-store-connect/#/devd274dd925)%1B[0m)
[09:52:11]: 🚫 Error: AppStoreConnect/screenshots/es-MX/5_iphone67_5.AppStoreViewClipping.png - Invalid screen size (Actual size is 1290x2796. See the specifications to fix [https://help.apple.com/app-store-connect/#/devd274dd925)](https://help.apple.com/app-store-connect/#/devd274dd925)%1B[0m)
[09:52:11]: 🚫 Error: AppStoreConnect/screenshots/es-MX/6_iphone67_6.AppStoreWikipedia.png - Invalid screen size (Actual size is 1290x2796. See the specifications to fix [https://help.apple.com/app-store-connect/#/devd274dd925)](https://help.apple.com/app-store-connect/#/devd274dd925)%1B[0m)
[09:52:11]: 🚫 Error: AppStoreConnect/screenshots/fr-FR/1_iphone67_1.AppStoreNihongoEntry.png - Invalid screen size (Actual size is 1290x2796. See the specifications to fix [https://help.apple.com/app-store-connect/#/devd274dd925)](https://help.apple.com/app-store-connect/#/devd274dd925)%1B[0m)
[09:52:11]: 🚫 Error: AppStoreConnect/screenshots/fr-FR/2_iphone67_2.AppStoreKanjiFlashcard.png - Invalid screen size (Actual size is 1290x2796. See the specifications to fix [https://help.apple.com/app-store-connect/#/devd274dd925)](https://help.apple.com/app-store-connect/#/devd274dd925)%1B[0m)
[09:52:11]: 🚫 Error: AppStoreConnect/screenshots/fr-FR/3_iphone67_3.AppStoreWordTriage.png - Invalid screen size (Actual size is 1290x2796. See the specifications to fix [https://help.apple.com/app-store-connect/#/devd274dd925)](https://help.apple.com/app-store-connect/#/devd274dd925)%1B[0m)
[09:52:11]: 🚫 Error: AppStoreConnect/screenshots/fr-FR/4_iphone67_4.AppStoreOCR.png - Invalid screen size (Actual size is 1290x2796. See the specifications to fix [https://help.apple.com/app-store-connect/#/devd274dd925)](https://help.apple.com/app-store-connect/#/devd274dd925)%1B[0m)
[09:52:11]: 🚫 Error: AppStoreConnect/screenshots/fr-FR/5_iphone67_5.AppStoreViewClipping.png - Invalid screen size (Actual size is 1290x2796. See the specifications to fix [https://help.apple.com/app-store-connect/#/devd274dd925)](https://help.apple.com/app-store-connect/#/devd274dd925)%1B[0m)
[09:52:11]: 🚫 Error: AppStoreConnect/screenshots/fr-FR/6_iphone67_6.AppStoreWikipedia.png - Invalid screen size (Actual size is 1290x2796. See the specifications to fix [https://help.apple.com/app-store-connect/#/devd274dd925)](https://help.apple.com/app-store-connect/#/devd274dd925)%1B[0m)
[09:52:11]: 🚫 Error: AppStoreConnect/screenshots/nl-NL/1_iphone67_1.AppStoreNihongoEntry.png - Invalid screen size (Actual size is 1290x2796. See the specifications to fix [https://help.apple.com/app-store-connect/#/devd274dd925)](https://help.apple.com/app-store-connect/#/devd274dd925)%1B[0m)
[09:52:11]: 🚫 Error: AppStoreConnect/screenshots/nl-NL/2_iphone67_2.AppStoreKanjiFlashcard.png - Invalid screen size (Actual size is 1290x2796. See the specifications to fix [https://help.apple.com/app-store-connect/#/devd274dd925)](https://help.apple.com/app-store-connect/#/devd274dd925)%1B[0m)
[09:52:11]: 🚫 Error: AppStoreConnect/screenshots/nl-NL/3_iphone67_3.AppStoreWordTriage.png - Invalid screen size (Actual size is 1290x2796. See the specifications to fix [https://help.apple.com/app-store-connect/#/devd274dd925)](https://help.apple.com/app-store-connect/#/devd274dd925)%1B[0m)
[09:52:11]: 🚫 Error: AppStoreConnect/screenshots/nl-NL/4_iphone67_4.AppStoreOCR.png - Invalid screen size (Actual size is 1290x2796. See the specifications to fix [https://help.apple.com/app-store-connect/#/devd274dd925)](https://help.apple.com/app-store-connect/#/devd274dd925)%1B[0m)
[09:52:11]: 🚫 Error: AppStoreConnect/screenshots/nl-NL/5_iphone67_5.AppStoreViewClipping.png - Invalid screen size (Actual size is 1290x2796. See the specifications to fix [https://help.apple.com/app-store-connect/#/devd274dd925)](https://help.apple.com/app-store-connect/#/devd274dd925)%1B[0m)
[09:52:11]: 🚫 Error: AppStoreConnect/screenshots/nl-NL/6_iphone67_6.AppStoreWikipedia.png - Invalid screen size (Actual size is 1290x2796. See the specifications to fix [https://help.apple.com/app-store-connect/#/devd274dd925)](https://help.apple.com/app-store-connect/#/devd274dd925)%1B[0m)
[09:52:11]: 🚫 Error: AppStoreConnect/screenshots/ru/1_iphone67_1.AppStoreNihongoEntry.png - Invalid screen size (Actual size is 1290x2796. See the specifications to fix [https://help.apple.com/app-store-connect/#/devd274dd925)](https://help.apple.com/app-store-connect/#/devd274dd925)%1B[0m)
[09:52:11]: 🚫 Error: AppStoreConnect/screenshots/ru/2_iphone67_2.AppStoreKanjiFlashcard.png - Invalid screen size (Actual size is 1290x2796. See the specifications to fix [https://help.apple.com/app-store-connect/#/devd274dd925)](https://help.apple.com/app-store-connect/#/devd274dd925)%1B[0m)
[09:52:11]: 🚫 Error: AppStoreConnect/screenshots/ru/3_iphone67_3.AppStoreWordTriage.png - Invalid screen size (Actual size is 1290x2796. See the specifications to fix [https://help.apple.com/app-store-connect/#/devd274dd925)](https://help.apple.com/app-store-connect/#/devd274dd925)%1B[0m)
[09:52:11]: 🚫 Error: AppStoreConnect/screenshots/ru/4_iphone67_4.AppStoreOCR.png - Invalid screen size (Actual size is 1290x2796. See the specifications to fix [https://help.apple.com/app-store-connect/#/devd274dd925)](https://help.apple.com/app-store-connect/#/devd274dd925)%1B[0m)
[09:52:11]: 🚫 Error: AppStoreConnect/screenshots/ru/5_iphone67_5.AppStoreViewClipping.png - Invalid screen size (Actual size is 1290x2796. See the specifications to fix [https://help.apple.com/app-store-connect/#/devd274dd925)](https://help.apple.com/app-store-connect/#/devd274dd925)%1B[0m)
[09:52:11]: 🚫 Error: AppStoreConnect/screenshots/ru/6_iphone67_6.AppStoreWikipedia.png - Invalid screen size (Actual size is 1290x2796. See the specifications to fix [https://help.apple.com/app-store-connect/#/devd274dd925)](https://help.apple.com/app-store-connect/#/devd274dd925)%1B[0m)
[09:52:11]: 🚫 Error: AppStoreConnect/screenshots/zh-Hans/1_iphone67_1.AppStoreNihongoEntry.png - Invalid screen size (Actual size is 1290x2796. See the specifications to fix [https://help.apple.com/app-store-connect/#/devd274dd925)](https://help.apple.com/app-store-connect/#/devd274dd925)%1B[0m)
[09:52:11]: 🚫 Error: AppStoreConnect/screenshots/zh-Hans/2_iphone67_2.AppStoreKanjiFlashcard.png - Invalid screen size (Actual size is 1290x2796. See the specifications to fix [https://help.apple.com/app-store-connect/#/devd274dd925)](https://help.apple.com/app-store-connect/#/devd274dd925)%1B[0m)
[09:52:11]: 🚫 Error: AppStoreConnect/screenshots/zh-Hans/3_iphone67_3.AppStoreWordTriage.png - Invalid screen size (Actual size is 1290x2796. See the specifications to fix [https://help.apple.com/app-store-connect/#/devd274dd925)](https://help.apple.com/app-store-connect/#/devd274dd925)%1B[0m)
[09:52:11]: 🚫 Error: AppStoreConnect/screenshots/zh-Hans/4_iphone67_4.AppStoreOCR.png - Invalid screen size (Actual size is 1290x2796. See the specifications to fix [https://help.apple.com/app-store-connect/#/devd274dd925)](https://help.apple.com/app-store-connect/#/devd274dd925)%1B[0m)
[09:52:11]: 🚫 Error: AppStoreConnect/screenshots/zh-Hans/5_iphone67_5.AppStoreViewClipping.png - Invalid screen size (Actual size is 1290x2796. See the specifications to fix [https://help.apple.com/app-store-connect/#/devd274dd925)](https://help.apple.com/app-store-connect/#/devd274dd925)%1B[0m)
[09:52:11]: 🚫 Error: AppStoreConnect/screenshots/zh-Hans/6_iphone67_6.AppStoreWikipedia.png - Invalid screen size (Actual size is 1290x2796. See the specifications to fix [https://help.apple.com/app-store-connect/#/devd274dd925)](https://help.apple.com/app-store-connect/#/devd274dd925)%1B[0m)
[09:52:11]: 🚫 Error: AppStoreConnect/screenshots/zh-Hant/1_iphone67_1.AppStoreNihongoEntry.png - Invalid screen size (Actual size is 1290x2796. See the specifications to fix [https://help.apple.com/app-store-connect/#/devd274dd925)](https://help.apple.com/app-store-connect/#/devd274dd925)%1B[0m)
[09:52:11]: 🚫 Error: AppStoreConnect/screenshots/zh-Hant/2_iphone67_2.AppStoreKanjiFlashcard.png - Invalid screen size (Actual size is 1290x2796. See the specifications to fix [https://help.apple.com/app-store-connect/#/devd274dd925)](https://help.apple.com/app-store-connect/#/devd274dd925)%1B[0m)
[09:52:11]: 🚫 Error: AppStoreConnect/screenshots/zh-Hant/3_iphone67_3.AppStoreWordTriage.png - Invalid screen size (Actual size is 1290x2796. See the specifications to fix [https://help.apple.com/app-store-connect/#/devd274dd925)](https://help.apple.com/app-store-connect/#/devd274dd925)%1B[0m)
[09:52:11]: 🚫 Error: AppStoreConnect/screenshots/zh-Hant/4_iphone67_4.AppStoreOCR.png - Invalid screen size (Actual size is 1290x2796. See the specifications to fix [https://help.apple.com/app-store-connect/#/devd274dd925)](https://help.apple.com/app-store-connect/#/devd274dd925)%1B[0m)
[09:52:11]: 🚫 Error: AppStoreConnect/screenshots/zh-Hant/5_iphone67_5.AppStoreViewClipping.png - Invalid screen size (Actual size is 1290x2796. See the specifications to fix [https://help.apple.com/app-store-connect/#/devd274dd925)](https://help.apple.com/app-store-connect/#/devd274dd925)%1B[0m)
[09:52:11]: 🚫 Error: AppStoreConnect/screenshots/zh-Hant/6_iphone67_6.AppStoreWikipedia.png - Invalid screen size (Actual size is 1290x2796. See the specifications to fix [https://help.apple.com/app-store-connect/#/devd274dd925)](https://help.apple.com/app-store-connect/#/devd274dd925)%1B[0m)

[!] Canceled uploading screenshots. Please check the error messages above and fix the screenshots.

</details>

### Environment

<details><summary>✅ fastlane environment ✅</summary>

### Stack

| Key                         | Value                                                                                |
| --------------------------- | ------------------------------------------------------------------------------------ |
| OS                          | 12.5.1                                                                               |
| Ruby                        | 3.0.1                                                                                |
| Bundler?                    | true                                                                                 |
| Git                         | git version 2.37.0 (Apple Git-136)                                                   |
| Installation Source         | ~/jenkins/workspace/Fastlane Env/StudyJapanese/vendor/bundle/ruby/3.0.0/bin/fastlane |
| Host                        | macOS 12.5.1 (21G83)                                                                 |
| Ruby Lib Dir                | ~/.rbenv/versions/3.0.1/lib                                                          |
| OpenSSL Version             | OpenSSL 1.1.1j  16 Feb 2021                                                          |
| Is contained                | false                                                                                |
| Is homebrew                 | false                                                                                |
| Is installed via Fabric.app | false                                                                                |
| Xcode Path                  | /Applications/Xcode.app/Contents/Developer/                                          |
| Xcode Version               | 14.0                                                                                 |
| Swift Version               | 5.7                                                                                  |

### System Locale

| Variable | Value       |   |
| -------- | ----------- | - |
| LANG     | en_US.UTF-8 | ✅ |
| LC_ALL   | en_US.UTF-8 | ✅ |
| LANGUAGE |             |   |

### fastlane files:

**No Fastfile found**

**No Appfile found**

### fastlane gems

| Gem      | Version | Update-Status |
| -------- | ------- | ------------- |
| fastlane | 2.209.1 | ✅ Up-To-Date  |
| trainer  | 0.9.1   | ✅ Up-To-Date  |

### Loaded fastlane plugins:

**No plugins Loaded**

<details><summary><b>Loaded gems</b></summary>

| Gem                             | Version      |
| ------------------------------- | ------------ |
| did_you_mean                    | 1.5.0        |
| bundler                         | 2.2.15       |
| tsort                           | 0.1.0        |
| uri                             | 0.10.1       |
| pathname                        | 0.1.0        |
| rake                            | 13.0.6       |
| rexml                           | 3.2.5        |
| CFPropertyList                  | 3.0.5        |
| concurrent-ruby                 | 1.1.10       |
| i18n                            | 1.12.0       |
| minitest                        | 5.16.2       |
| tzinfo                          | 2.0.5        |
| zeitwerk                        | 2.6.0        |
| activesupport                   | 6.1.6.1      |
| public_suffix                   | 4.0.7        |
| addressable                     | 2.8.1        |
| httpclient                      | 2.8.3        |
| json                            | 2.6.2        |
| algoliasearch                   | 1.27.5       |
| artifactory                     | 3.0.15       |
| atomos                          | 0.1.3        |
| aws-eventstream                 | 1.2.0        |
| aws-partitions                  | 1.629.0      |
| aws-sigv4                       | 1.5.1        |
| jmespath                        | 1.6.1        |
| aws-sdk-core                    | 3.146.0      |
| aws-sdk-kms                     | 1.58.0       |
| aws-sdk-s3                      | 1.114.0      |
| babosa                          | 1.0.4        |
| claide                          | 1.1.0        |
| fuzzy_match                     | 2.0.4        |
| nap                             | 1.1.0        |
| netrc                           | 0.11.0       |
| ffi                             | 1.15.5       |
| ethon                           | 0.15.0       |
| typhoeus                        | 1.4.0        |
| cocoapods-core                  | 1.11.3       |
| cocoapods-deintegrate           | 1.0.5        |
| cocoapods-downloader            | 1.6.3        |
| cocoapods-plugins               | 1.0.0        |
| cocoapods-search                | 1.0.1        |
| cocoapods-trunk                 | 1.6.0        |
| cocoapods-try                   | 1.2.0        |
| colored2                        | 3.1.2        |
| escape                          | 0.0.4        |
| fourflusher                     | 2.3.1        |
| gh_inspector                    | 1.1.3        |
| molinillo                       | 0.8.0        |
| ruby-macho                      | 2.5.1        |
| nanaimo                         | 0.3.0        |
| xcodeproj                       | 1.22.0       |
| cocoapods                       | 1.11.3       |
| colored                         | 1.2          |
| highline                        | 2.0.3        |
| commander                       | 4.6.0        |
| declarative                     | 0.0.20       |
| digest-crc                      | 0.6.4        |
| unf_ext                         | 0.0.8.2      |
| unf                             | 0.1.4        |
| domain_name                     | 0.5.20190701 |
| dotenv                          | 2.8.1        |
| emoji_regex                     | 3.2.3        |
| excon                           | 0.92.4       |
| faraday-em_http                 | 1.0.0        |
| faraday-em_synchrony            | 1.0.0        |
| faraday-excon                   | 1.1.0        |
| faraday-httpclient              | 1.0.1        |
| multipart-post                  | 2.0.0        |
| faraday-multipart               | 1.0.4        |
| faraday-net_http                | 1.0.1        |
| faraday-net_http_persistent     | 1.2.0        |
| faraday-patron                  | 1.0.0        |
| faraday-rack                    | 1.0.0        |
| faraday-retry                   | 1.0.3        |
| ruby2_keywords                  | 0.0.5        |
| faraday                         | 1.10.2       |
| http-cookie                     | 1.0.5        |
| faraday-cookie_jar              | 0.0.7        |
| faraday_middleware              | 1.2.0        |
| fastimage                       | 2.2.6        |
| jwt                             | 2.5.0        |
| memoist                         | 0.16.2       |
| multi_json                      | 1.15.0       |
| os                              | 1.1.4        |
| signet                          | 0.17.0       |
| googleauth                      | 1.2.0        |
| mini_mime                       | 1.1.2        |
| trailblazer-option              | 0.1.2        |
| uber                            | 0.1.0        |
| representable                   | 3.2.0        |
| retriable                       | 3.1.2        |
| webrick                         | 1.7.0        |
| google-apis-core                | 0.7.0        |
| google-apis-androidpublisher_v3 | 0.26.0       |
| google-apis-playcustomapp_v1    | 0.10.0       |
| google-apis-iamcredentials_v1   | 0.13.0       |
| google-apis-storage_v1          | 0.17.0       |
| google-cloud-env                | 1.6.0        |
| google-cloud-errors             | 1.3.0        |
| google-cloud-core               | 1.6.0        |
| google-cloud-storage            | 1.39.0       |
| mini_magick                     | 4.11.0       |
| naturally                       | 2.2.1        |
| optparse                        | 0.1.1        |
| plist                           | 3.6.0        |
| rubyzip                         | 2.3.2        |
| security                        | 0.1.3        |
| simctl                          | 1.6.8        |
| terminal-notifier               | 2.0.0        |
| unicode-display_width           | 1.8.0        |
| terminal-table                  | 1.8.0        |
| tty-screen                      | 0.8.1        |
| tty-cursor                      | 0.7.1        |
| tty-spinner                     | 0.9.3        |
| word_wrap                       | 1.0.0        |
| rouge                           | 2.0.7        |
| xcpretty                        | 0.3.0        |
| xcpretty-travis-formatter       | 1.0.1        |
</details>

*generated on:* **2022-09-14**
</details>

### PR DESCRIPTION
**Pull request by cherpake**
### Checklist
- [x] I've run `bundle exec rspec` from the root directory to see all new and existing tests pass
- [x] I've followed the _fastlane_ code style and run `bundle exec rubocop -a` to ensure the code style is valid
- [x] I see several green `ci/circleci` builds in the "All checks have passed" section of my PR ([connect CircleCI to GitHub](https://support.circleci.com/hc/en-us/articles/360008097173-Why-aren-t-pull-requests-triggering-jobs-on-my-organization-) if not)
- [x] I've read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I've updated the documentation if necessary.

### Description
Added new screenshot sizes for 6.7inch iPhone 14 Pro Max

### CODE REVIEW

File: deliver/spec/app_screenshot_spec.rb [hunk 1]
```diff
@@ -308,6 +318,11 @@ def app_screenshot_with(screen_size, path = '', language = 'en-US')
       expect(app_screenshot_with(ScreenSize::IOS_61).device_type).to be_nil
     end
 
+    it "should return iphone65 for 6.7 inch displays" do
```
Comment (rogerluan) [inline deliver/spec/app_screenshot_spec.rb:321]:
```suggestion
    it "should return iphone67 for 6.7 inch displays" do
```

or 

```suggestion
    it "should return iphone14ProMax for 6.7 inch displays" do
```

🤔

File: fastlane/lib/fastlane/version.rb [hunk 2]
```diff
@@ -1,5 +1,5 @@
 module Fastlane
-  VERSION = '2.210.1'.freeze
+  VERSION = '2.210.2'.freeze
```
Comment (rogerluan) [inline fastlane/lib/fastlane/version.rb:2]: Please revert this change as it shouldn't be part of a PR 😊 this gets updated during deployment processes

**Review by rogerluan — COMMENTED** [review]
Thank you for this, @cherpake ! Have you confirmed it works by uploading 6.7" screenshots to the App Store?

Would you mind signing the CLA with Google? The instructions were posted by the bot, right above :)

Thanks once again!

**Review by rogerluan — CHANGES_REQUESTED** [review]
Could you also check for linter issues by running `bundle exec rubocop -a`? 🙏

**Review by giginet — COMMENTED** [review]
I just need this feature. So I'll try this patch. Good job ✨ 

LGTM except for existing comments.
Could you fix them?

**Comment by cherpake** [comment]
> 

Done, fixed one issue

File: Gemfile.lock [hunk 3]
```diff
@@ -1,7 +1,7 @@
 PATH
   remote: .
   specs:
-    fastlane (2.210.1)
+    fastlane (2.210.2)
```
Comment (giginet) [inline Gemfile.lock:4]: Could you revert `Gemfile.lock`?
This file should not be changed by this PR.

File: deliver/lib/deliver/app_screenshot.rb [hunk 4]
```diff
@@ -187,6 +196,10 @@ def self.device_messages
       # This list does not include iPad Pro 12.9-inch (3rd generation)
       # because it has same resoluation as IOS_IPAD_PRO and will clobber
       return {
+        ScreenSize::IOS_67_MESSAGES => [
+          [1290, 2796],
+          [2796, 1290]
```
Comment (giginet) [inline deliver/lib/deliver/app_screenshot.rb:201]: You can add a trailing comma for conventions.

```suggestion
          [2796, 1290],
```

File: deliver/lib/deliver/app_screenshot.rb [hunk 5]
```diff
@@ -247,6 +260,10 @@ def self.devices
       # This list does not include iPad Pro 12.9-inch (3rd generation)
       # because it has same resoluation as IOS_IPAD_PRO and will clobber
       return {
+         ScreenSize::IOS_67 => [
+           [1290, 2796],
+           [2796, 1290]
```
Comment (giginet) [inline deliver/lib/deliver/app_screenshot.rb:265]:
```suggestion
           [2796, 1290],
```

File: deliver/spec/app_screenshot_spec.rb [hunk 6]
```diff
@@ -148,6 +153,11 @@ def expect_screen_size_from_file(file)
     end
 
     describe "valid iMessage app screen sizes" do
+      it "should calculate all 6.5 inch iPhone resolutions" do
```
Comment (giginet) [inline deliver/spec/app_screenshot_spec.rb:156]:
```suggestion
      it "should calculate all 6.7 inch iPhone resolutions" do
```

**Review by giginet — CHANGES_REQUESTED** [review]
@cherpake 
Could you revert the changes for `Gemfile.lock`?

And I pointed out minor fixes.

After you fixed, it will be merged soon.

**Comment by giginet** [comment]
@cherpake  I'm so sorry for your inconvenience 🙏  our CI avoided placing trailing commas.
So my suggestion is completely wrong 🙇 
Could you revert them? 

https://github.com/fastlane/fastlane/pull/20694/commits/673e1b908f804554d0f182a28bf07e7aa4bd856c
https://github.com/fastlane/fastlane/pull/20694/commits/a758f8aa2bf41699ad9d7f2dfe4dcaa011e11ce9

File: deliver/lib/deliver/app_screenshot.rb [hunk 7]
```diff
@@ -187,6 +196,10 @@ def self.device_messages
       # This list does not include iPad Pro 12.9-inch (3rd generation)
       # because it has same resoluation as IOS_IPAD_PRO and will clobber
       return {
+        ScreenSize::IOS_67_MESSAGES => [
+          [1290, 2796],
+          [2796, 1290],
```
Comment (giginet) [inline deliver/lib/deliver/app_screenshot.rb:201]: My suggestion was wrong 🙇 

```suggestion
          [2796, 1290]
```

File: deliver/lib/deliver/app_screenshot.rb [hunk 8]
```diff
@@ -247,6 +260,10 @@ def self.devices
       # This list does not include iPad Pro 12.9-inch (3rd generation)
       # because it has same resoluation as IOS_IPAD_PRO and will clobber
       return {
+         ScreenSize::IOS_67 => [
+           [1290, 2796],
+           [2796, 1290],
```
Comment (giginet) [inline deliver/lib/deliver/app_screenshot.rb:265]:
```suggestion
           [2796, 1290]
```

**Comment by cherpake** [comment]
I want Fastlane t-shirt for this :)

**Comment by giginet** [comment]
Could you revert Gemfile.lock?
https://github.com/fastlane/fastlane/pull/20694/files#r984199848

You just execute `bundle install`.

**Comment by cherpake** [comment]
Done...

**Review by giginet — APPROVED** [review]
LGTM. Thank you for your contribution 😄 
I'll merge this after checking with other members.

File: deliver/lib/deliver/app_screenshot.rb [hunk 9]
```diff
@@ -247,6 +260,10 @@ def self.devices
       # This list does not include iPad Pro 12.9-inch (3rd generation)
       # because it has same resoluation as IOS_IPAD_PRO and will clobber
       return {
+         ScreenSize::IOS_67 => [
+           [1290, 2796],
+           [2796, 1290]
+         ],
```
Comment (rogerluan) [inline deliver/lib/deliver/app_screenshot.rb:266]:
```suggestion
        ScreenSize::IOS_67 => [
          [1290, 2796],
          [2796, 1290]
        ],
```

**Review by rogerluan — CHANGES_REQUESTED** [review]

**Review by rogerluan — APPROVED** [review]
🚀 LGTM! Thank you again for this PR @cherpake ! 
(and I wish we had fastlane T-shirts too! ❤️ )

**Pull request merged**